website/blog/2021-01/01/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Happy anniversary Gardener! Three years of open source Kubernetes management
+linkTitle: Happy anniversary Gardener! Three years of open source Kubernetes management
+newsSubtitle: January 30, 2021
+type: blog
+publishdate: 2021-01-25
+archivedate: 2021-02-25
+authors:
+- name: Vasu Chandrasekhara
+  email: vasu.chandrasekhara@sap.com
+  avatar: https://avatars.githubusercontent.com/u/11454395?s=400&u=9496275f0718d81bea32068dccb4d9c2d848e592&v=4
+- name: Tim Usner
+  email: tim.usner@sap.com
+  avatar: https://avatars.githubusercontent.com/u/40451181?s=400&u=4df34635cf86b924700ef4152ec8462eeaa35721&v=4
+aliases: ["/blog/2021/01/30/01"]
```

Comment:
This path is supposed to reflect the publish date of the blog (formatted: `/blog/YYYY/MM/DD/index`). The final piece is the index at that date in case there have been multiple blogs issued on the same day.

Reply:
How am I supposed to know (right now) the publishing date and if there will be multiple articles being published on that date? The date this PR gets merged and the changes deployed can hardly be influenced from the contributors side.

Reply:
Your reviewer will know and prompt you to update accordingly if necessary. The case with multiple blogs per day is rare and only theoretically possible and your reviewer will control that too before giving you a go. Alias is just optional  convenience alternate path and can be added later too. 
In fact, `publishdate` is only optional and is useful when you prepare material in advance and want to have it in a publishing pipeline on a date in future.

Reply:
Okay, I removed `publishdate `.